what4/src/What4/Protocol/SMTWriter.hs

```diff
@@ -873,6 +873,12 @@ class (SupportTermOps (Term h)) => SMTWriter h where
   -- | Pop 1 existing scope
   popCommand    :: f h -> Command h
 
+  -- | Push 2 new scope
+  push2Command   :: f h -> Command h
+
+  -- | Pop 2 existing scope
+  pop2Command    :: f h -> Command h
```

Comment:
Can you elaborate a bit on the semantics of these? What smtlib is generated for these?

Reply:
Seems as descriptive as the [previous](https://github.com/GaloisInc/what4/blob/99a6808c0ebc0907e1ec584a5089e72de18dd4fc/what4/src/What4/Protocol/SMTWriter.hs#L870) two commands. I do explain [here](https://github.com/GaloisInc/what4/blob/99a6808c0ebc0907e1ec584a5089e72de18dd4fc/what4/src/What4/Protocol/Online.hs#L356), but I can move/copy this if you think it might be more appropriate in SMTWriter.